src/main/java/org/myrobotlab/service/JMonkeyEngine.java

```diff
@@ -2157,7 +2157,7 @@ public void simpleInitApp() {
     new File(getDataDir()).mkdirs();
     new File(getResourceDir()).mkdirs();
 
-    // assetManager.registerLocator("./", FileLocator.class);
+    assetManager.registerLocator("./", FileLocator.class);

```

Comment:
is there a way to do this without a relative path?  i worry 2 things.. 1 the current working directory might change.. also , 2 that this won't work on windows.